projects/plugins/jetpack/sal/class.json-api-post-base.php

```diff
@@ -596,7 +939,7 @@ private function get_media_item_v1_1( $media_id ) {
 				 * @since 3.9.0
 				 *
 				 * @param array $metadata['sizes'] Array of thumbnail sizes available for a given attachment ID.
-				 * @param string $media_id Attachment ID.
+				 * @param int   $media_id The media item ID.
```

Comment:
I've changed $media_id to be an int here, but it was previously added as a string. From what I can tell in testing, it’s an int (`get_media_type` takes an int for example and we use that function here). Perhaps there is some other information I'm missing though so I wanted to highlight this just in case.